src/index.html

```diff
@@ -1,13 +1,338 @@
+                      alt="headphones" 
+                      class="products__photo"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="products__description">
+              Headphones & earbuds
+            </div>
+          </div>
+          <div class="products">
+            <div class="products__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--tablet--1-4 grid__item--1-8">
+                  <div class="products__photo-container">
+                    <img
+                      src="./images/photos/speakers1.jpg"
+                      alt="speakers"
+                      class="products__photo"
+                    >
+                  </div>
+                </div>
+                <div class="grid__item grid__item--tablet--5-6 grid__item--9-12">
+                  <div class="products__photo-container">
+                    <img
+                      src="./images/photos/speakers2.jpg"
+                      alt="speakers"
+                      class="products__photo"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="products__description">
+              Speakers
+            </div>
+          </div>
+          <div class="products">
+            <div class="products__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--tablet--1-2 grid__item--1-4">
+                  <div class="products__photo-container">
+                    <img
+                      src="./images/photos/sunglasses1.jpg"
+                      alt="sunglasses"
+                      class="products__photo"
+                    >
+                  </div>
+                </div>
+                <div class="grid__item grid__item--tablet--3-6 grid__item--5-12">
+                  <div class="products__photo-container">
+                    <img
+                      src="./images/photos/sunglasses2.jpg"
+                      alt="sunglasses"
+                      class="products__photo"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="products__description">
+              Audio sunglasses
+            </div>
+          </div>
+
+          <div class="grid grid--mobile">
+            <div class="grid__item grid__item--5-8 grid__item--tablet--2-5">
+              <a href="#" class="button">
+                View all
+              </a>
+            </div>
+          </div>
+
+        </div>
+      </section>
+      <section class="page__section" id="about-us">
+        <div class="container">
+          <div class="grid grid--mobile">
+            <div class="grid__item grid__item--1-5 grid__item--tablet--1-3">
+              <h2 class="page__section-title">
```

Comment:
Due to Figma on Desktop this title sholudn't have text-align center, check your css code
https://prnt.sc/GIj3sSBs0v9L